website/content/docs/secrets/pki/considerations.mdx

```diff
@@ -217,6 +297,71 @@ nature:
  - `pem_bundle` this request parameter is only used on the issuer-import
    paths and may contain sensitive private key material.
 
+## Role-Based Access
+
+Vault supports [path-based ACL Policies](https://learn.hashicorp.com/tutorials/vault/getting-started-policies)
+for limiting access to various paths within Vault.
+
+The following is a condensed example reference of ACLing the PKI Secrets
+Engine. These are just a suggestion; other personas and policy approaches
+may also be valid.
+
+We suggest the following personas:
+
+ - *Operator*;  a privileged user who manages the health of the PKI
+   subsystem; manages issuers and key material.
+ - *Agent*; a semi-privileged that manages roles and handles
+   revocation on behalf of an operator; may also handle delegated
+   issuance.
+ - *Advanced*; potentially a power-user that has access to additional
+   issuance APIs.
+ - *Requester*; a low-level user that simply requests certificates.
+ - *Unauthed*; any arbitrary user or service that lacks a Vault token.
+
+For these personas, we suggest the following ACLs, in condensed, tabular form:
+
+| Path | Operations | Operator | Agent | Advanced | Requester | Unauthed |
```

Comment:
@ivana-mcconnell made a comment out of band that these might be good to have per-persona; currently they're sorted as such (so all the persona's paths are grouped together and you can then read across the table to find exactly those paths), but I'm not sure if it makes sense expanding these per-persona explicitly (duplicating endpoints across various personas). Does anyone know if we can do those fancy tables like on the tutorials pages (which I don't think are hosted on this repo)? 

This present table makes the additive nature visually clear, but sadly it is already too long to fit on a single (horizontal) screen, so maybe it doesn't matter and breaking it up would be more useful in general.

Should we make a better effort to make these in an ACL-able format (i.e., remove the regex patterns)? Or should we explicitly leave them in regex-style paths, to force people to think about the ACL actions they're potentially choosing here, as they are in general, risky?

Thoughts?

Reply:
This is really nice, but as you say quite long sadly. It's even missing some paths :( Pre-canned policies that enforce these personas might be interesting?

issuer/:issuer_ref/sign-intermediate
issuer/:issuer_ref/sign-self-issued
root/rotate/:exported

Reply:
The concern with pre-canned policies in particular is that you might rarely want to actually open up _all_ users to issue against _all_ roles; I think more securely, we'd recommend limiting users to specific roles, which is up to the operator to design and implement.

That alone makes the pre-canned approach harder, as we're just doing the "easy" (operator/agent) side and leaving the (advanced/requester) roles more broad.

Reply:
Fair, but we could have a pre-canned Requester policy with <:role_name:> or something in that regard to push them down that path?